flang/lib/Lower/OpenMP/OpenMP.cpp

```diff
@@ -946,13 +946,9 @@ genEnterExitUpdateDataOp(Fortran::lower::AbstractConverter &converter,
     cp.processMap(currentLocation, directive, stmtCtx, mapOperands);
   }
 
-  return firOpBuilder.create<OpTy>(
-      currentLocation, ifClauseOperand, deviceOperand,
-      dependTypeOperands.empty()
-          ? nullptr
-          : mlir::ArrayAttr::get(converter.getFirOpBuilder().getContext(),
-                                 dependTypeOperands),
-      dependOperands, nowaitAttr, mapOperands);
+  return firOpBuilder.create<OpTy>(currentLocation, ifClauseOperand,
+                                   deviceOperand, nullptr, mlir::ValueRange(),
```

Comment:
Nit: Add comment to document what these default values refer to.
```suggestion
                                   deviceOperand, /*depends=*/ nullptr, /*depend_vars=*/ mlir::ValueRange(),
```